elastic/kibana
# [Infra] Amend inventory title to "Infrastructure Inventory"

## Summary

Update the Inventory title terminology to be "Infrastructure Inventory", in order to differentiate from the incoming top-level "Inventory" page.

<img width="481" alt="Screenshot 2024-09-17 170745" src="https://github.com/user-attachments/assets/d72dfb6e-fe3b-44d8-b6f0-ae333586ceb4">

Closes #192324 

# How to test

- Load Elastic cluster, go to Infrastructure -> Infrastructure Inventory

**Expected**: The side menu link text should match the breadcrumb title and the main title text for the page.
